src/components/Card/Card.js

```diff
@@ -1,5 +1,85 @@
-const Card = () => {
-  return <div>Card</div>;
+import styled from 'styled-components';
+import { theme } from '../../styles/theme';
+
+const Card = ({ card }) => {
+  const { thumnail, categoryTag, category, title, price } = card;
+
+  const Category_Color = [
+    { type: '오리지널 VOD', color: `${theme.purple}` },
+    { type: '오프라인', color: `${theme.red}` },
+    { type: '신규오픈', color: `${theme.green}` },
+  ];
+
+  const bgColor = Category_Color.find(list => list.type === categoryTag).color;
```

Comment:
```suggestion
  const bgColor = Category_Color.find(({type}) => type === categoryTag).color;
```
- 구조분해할당 적용하면 조금 더 좋겠네요!! 별개로 상수데이터에는 color값이 들어가있지 않고 어떤 bg color를 설정할지는 styled components의 styling 부분에서 적용하는게 좀 더 관심사의 분리가 잘 된 코드같네요!! styled components가 type을 prop으로 받고 해당 prop을 이용해서 적절한 color를 매칭해주는식으로 처리하는게 좋겠네요! 이부분 이해 안되시면 오늘 라운지에서 찾아와주세요~!

Reply:
네 수정하였습니다!!